scripts/startup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# bash script to launch the
+# usage: bash scripts/startup.sh
+# this assumes you already ran: kind create cluster
+# Do not forget to add your RPC URL below
+export RPC_URL=""
+tilt down;
+tilt up;
+./mev exec alembic upgrade head;
+./mev prices fetch-all;
```

Comment:
You might want to use `&&` instead, as below:
```suggestion
tilt down &&
tilt up &&
./mev exec alembic upgrade head &&
./mev prices fetch-all
```

That way, if one of these command fails, it will not continue.